clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/SearchController.groovy

```diff
@@ -45,15 +48,28 @@ class SearchController {
    */
   @PreAuthorize("@fiatPermissionEvaluator.storeWholePermission()")
   @RequestMapping(value = '/search', method = RequestMethod.GET)
-  List<SearchResultSet> search(SearchQueryCommand q) {
+  List<SearchResultSet> search(@RequestParam(value = "q", defaultValue = "", required = false) String query,
```

Comment:
Setting a default value for `q` while marking as not required allows the wildcard search to still work correctly.  It's marked as not required because we will now allow for additional searching/filtering.  In deck, `q` basically becomes a keyword search in the new search field, e.g., `keyword:foo`.